concordium-sdk/src/main/java/com/concordium/sdk/transactions/account/IAccountTransaction2.java

```diff
@@ -0,0 +1,26 @@
+package com.concordium.sdk.transactions.account;
+
+import com.concordium.sdk.transactions.TransactionHeader;
+import com.concordium.sdk.transactions.TransactionSignature;
+import lombok.val;
+
+import java.nio.ByteBuffer;
+
+public abstract class IAccountTransaction2<T extends IAccountTransactionPayload2> {
```

Comment:
Plan is to have a Custom Builder for this Interface. 
With Methods like
`sign(pubKey, privKey)`
`setPayload(T payload)`